config.sample.yaml

```diff
@@ -168,6 +168,9 @@ thumbnails:
   # thumbnails, set this to false. If disabled, regular thumbnails will be returned.
   allowAnimated: true
 
+  # Default to animated thumbnails, if available
+  defaultAnimated: true
```

Comment:
I didn't mention it before (sorry), but to reduce surprises in clients the default should be `false` (given this feature is also missing a specification). I can make this change though.